src/libs/actions/WelcomeActions.js

```diff
@@ -68,7 +68,9 @@ function show({routes, toggleCreateMenu}) {
         // If we are rendering the SidebarScreen at the same time as a workspace route that means we've already created a workspace via workspace/new and should not open the global
         // create menu right now.
         const topRouteName = lodashGet(_.last(routes), 'name', '');
-        const isDisplayingWorkspaceRoute = topRouteName.toLowerCase().includes('workspace');
+        const loginWithShortLivedTokenRoute = _.find(routes, route => route.name === 'LogInWithShortLivedToken');
+        const exitingToWorkspaceRoute = lodashGet(loginWithShortLivedTokenRoute, 'params.exitTo', '') === 'workspace/new';
```

Comment:
Added this otherwise we would pop open the create menu in some contexts. I guess the top route is not always "workspace" and sometimes still the `LoginWithShortLivedToken` page.